src/box/memtx_space.c

```diff
@@ -1231,30 +1282,38 @@ memtx_build_on_replace(struct trigger *trigger, void *event)
 	 * problem when rollbacked changes appears in
 	 * built-in-background index.
 	 */
-	struct on_rollback_trigger_with_data *on_rollback_associates = NULL;
+	struct txn_triggers_with_data *txn_triggers = NULL;
 	struct errinj *inj = errinj(ERRINJ_BUILD_INDEX_ON_ROLLBACK_ALLOC,
 				    ERRINJ_BOOL);
 	if (inj == NULL || inj->bparam == false) {
-		on_rollback_associates = region_aligned_alloc(
+		txn_triggers = region_aligned_alloc(
```

Comment:
Let's use `xregion_aligned_alloc` here and drop the error injection.

Reply:
Done.